### PR TITLE
Features/vcf2cytosure

### PR DESCRIPTION
**Pull request by northwestwitch**

### CODE REVIEW

File: lib/MIP/Program/Variantcalling/Tiddit.pm [hunk 1]
```diff
@@ -25,13 +25,110 @@ BEGIN {
...
+    my $stdoutfile_path;
+
+    ## Default(s)
+    my $bin_size;
+
+    my $tmpl = {
+        bin_size => {
+            allow       => qr/ ^\d+$ /sxm,
+            default     => 500,
+            strict_type => 1,
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Tiddit.pm]: We have moved so far now to even order the keys in the template in AB, so place store above strict etc

File: lib/MIP/Program/Variantcalling/Vcf2cytosure.pm [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    ## Default(s)
+    my $frequency;
+    my $frequency_tag;
+    my $no_filter;
+    my $variant_size;
+
+    my $tmpl = {
+        coverage_file => {
+            required    => 1,
+            defined     => 1,
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Vcf2cytosure.pm]: Order AB for keys

File: t/tiddit_coverage.t [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+## to enable testing of each individual argument
+my %required_argument = (
+    infile_path => {
+        input           => q{infile_path},
+        expected_output => q{--bam infile_path},
+    },
+);
+
+my %specific_argument = (
+    bin_size => {
```
Comment (henrikstranneheim) [inline t/tiddit_coverage.t]: What about $outfile_path_prefix?

File: t/tiddit_coverage.t [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+## Test both base and function specific arguments
+my @arguments = ( \%base_argument, \%specific_argument );
+
+ARGUMENT_HASH_REF:
+foreach my $argument_href (@arguments) {
+    my @commands = test_function(
+        {
+            argument_href          => $argument_href,
+            required_argument_href => \%required_argument,
```
Comment (henrikstranneheim) [inline t/tiddit_coverage.t]: Order AB

**Comment by northwestwitch** [comment]
Yeah, I'll do the test also for the converter, I just wanted to send
something to review since I'll be late. I'll fix the rest as well!

Chiara

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**